projectJava/task2_calculator/src/CalculatorDL.java

```diff
@@ -0,0 +1,61 @@
+import java.util.Scanner;
+
+public class CalculatorDL {
+    static void executeCalculation(double num1, double num2, char operator) {
+        double result;
+        switch (operator) {
+            case '+':
```

Comment:
Please move arithmetic operators to constant, in case you are using it more than one time (here and below)